### DISCUSSION TITLE
unable to generate certificate

### DISCUSSION THREAD

**Discussion by lauhong28** [polymind-inc/acmebot#569] [Q&A] [answered]
I have followed the guide to setup acmebot with Azure DNS. All access granted accordingly however when i trying to go dashboard and generate certificate will receive this error message. 
##########
Orchestrator function 'IssueCertificate' failed: The activity function 'Dns01Precondition' failed: "The delegated name server is not correct. DNS zone = prd.dnstest.com, Expected = ns1-05.azure-dns.com,ns2-05.azure-dns.net,ns3-05.azure-dns.org,ns4-05.azure-dns.info, Actual = ". See the function execution logs for additional details.
############
May i know where i can check further about this issues. thanks in advance. 

![image](https://user-images.githubusercontent.com/32637333/225659968-da4f2ae8-c2ee-4ab7-a7a8-73dc793b3981.png)

**Comment by lauhong28** [comment]
i just saw another post have similar issue and understand this acemebot using Google DNS to resolve. i will  check again later. this dnstest.com just created yesterday for POC project.

**Comment by shibayan** [comment, marked as answer]
You need to delegate the domain's NS servers to Azure DNS. This has nothing to do with Acmebot, but with domains and DNS.
